cmd/init.go

```diff
@@ -109,90 +111,92 @@ var initCmd = &cli.Command{
 	Usage: "create a new gqlgen project",
 	Flags: []cli.Flag{
 		&cli.BoolFlag{Name: "verbose, v", Usage: "show logs"},
-		&cli.StringFlag{Name: "config, c", Usage: "the config filename"},
+		&cli.StringFlag{Name: "config, c", Usage: "the config filename", Value: "gqlgen.yml"},
 		&cli.StringFlag{Name: "server", Usage: "where to write the server stub to", Value: "server.go"},
 		&cli.StringFlag{Name: "schema", Usage: "where to write the schema stub to", Value: "graph/schema.graphqls"},
 	},
 	Action: func(ctx *cli.Context) error {
 		configFilename := ctx.String("config")
 		serverFilename := ctx.String("server")
+		schemaFilename := ctx.String("schema")
 
 		pkgName := code.ImportPathForDir(".")
 		if pkgName == "" {
 			return fmt.Errorf("unable to determine import path for current directory, you probably need to run go mod init first")
 		}
 
-		if err := initSchema(ctx.String("schema")); err != nil {
-			return err
+		// check schema and config don't already exist
+		if fileExists(configFilename) {
+			return fmt.Errorf("%s already exists\n", configFilename)
+		}
+		if fileExists(schemaFilename) {
+			return fmt.Errorf("%s already exists\n", schemaFilename)
 		}
-		if !configExists(configFilename) {
-			if err := initConfig(configFilename, pkgName); err != nil {
-				return err
-			}
+		if fileExists(serverFilename) {
+			return fmt.Errorf("%s already exists\n", serverFilename)
+		}
+		_, err := config.LoadConfigFromDefaultLocations()
+		if err == nil {
+			return fmt.Errorf("gqlgen.yml already exists in a parent directory\n")
 		}
```

Comment:
Some suggestions for improvement. Also, I think the `\n` belongs to the place the error is printed and should not be part of the message. 

```suggestion
		for _, name := range []string{configFilename, schemaFilename, serverFilename} {
			if fileExists(name) {
				return fmt.Errorf("%s already exists", name)
			}
		}
		if _, err := config.LoadConfigFromDefaultLocations(); err == nil {
			return errors.New("gqlgen.yml already exists in a parent directory")
		}
```

Reply:
hmm the newline is there because for some reason github.com/urfave/cli/v2 runs the error message together 🤷

```
$ go run github.com/99designs/gqlgen init
gqlgen.yml already existsexit status 1
```

Reply:
fixed in 9162c53fc32471c96d874764b47bf71f0c493fce